src/python/test/sugar/__init__.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import print_function, unicode_literals, division, absolute_import
+import contextlib
+import os
+import shutil
+import subprocess
+import tempfile
+from uuid import uuid4
+
+
+import logging
+logging.basicConfig(level="INFO")
```

Comment:
This is fine as is, just a preference thing on my end, change if you want to.

```suggestion
logging.basicConfig(level=logging.INFO)
```